templates/email/bexley/alert-update.txt

```diff
@@ -1,7 +1,12 @@
+[% is_missed_collection = report.category == 'Report missed collection' ~%]
+[%~ IF is_missed_collection ~%]
 Subject: New updates on your missed collection report
+[%~ ELSE ~%]
+Subject: New [% site_name %] updates on report: '[% report.title %]'
+[%~ END %]
 
-You asked us to send you an email every time an update was made to the
-[% site_name %] report: [% report.title %].
+You asked us to send you an email every time an update was made to
+[% IF is_missed_collection %]your missed collection report[% ELSE %]the [% site_name %] report: [% report.title %][% END %].
 
```

Comment:
I'm annoyed I let this slip through the cracks somehow, as the HTML counterpart file seems fine...Thanks for picking this up 🙏